extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnPageReaderImpl.java

```diff
@@ -264,16 +273,65 @@ private static Encoding getEncoding(final org.apache.parquet.format.Encoding enc
         return org.apache.parquet.column.Encoding.valueOf(encoding.name());
     }
 
-    private int readRowCountFromPageV1(final DataPageV1 page) {
+    /**
+     * Reads a little-endian integer from the provided {@link DataInput}.
+     *
+     * @throws IOException If an error occurs while reading from the input stream
+     * @implNote DataInput reads big-endian, so we need to reverse the bytes
+     */
+    private static int readLittleEndianInt(final DataInput in) throws IOException {
+        return Integer.reverseBytes(in.readInt());
+    }
+
+    /**
+     * Helper struct for holding a ByteBuffer.
+     */
+    // TODO Better name
```

Comment:
TODO

Reply:
Seems OK. Maybe `SizedByteBuffer` for symmetry?